python-package/Readme.rst

```diff
@@ -178,7 +178,7 @@ Known Issues
 
 * FastRGF crashes if training dataset is too small (#data < 28). (`rgf#92 <https://github.com/RGF-team/rgf/issues/92>`__)
```

Comment:
This line should be remained here?

Reply:
Yeah, I think that `rgf_python` users should know about it without the need to search in other places.